CreeDictionary/DatabaseManager/xml_importer.py

```diff
@@ -1,6 +1,4 @@
-import datetime
 import re
-import time
 import xml.etree.ElementTree as ET
 from collections import defaultdict
 from pathlib import Path
```

Comment:
unused import clean up. just accidental

Reply:
In principle and in practice, we wouldn't really want to use the old non-TSV layout files anymore, as all the information should be found in the *.layout.tsv files. On occasion, I've found working with the generated *.layout files useful in that the formatting becomes explicit, and you avoid certain spreadsheet application stupidities, such as the automatic conversion of plain quotes to "smart" (i.e. curly) quotes, if you don't watch out.

If our itwêwina code, reading the *.layout.tsv files, can interpret alike a plain and a curly quote, then that shouldn't be an issue anymore - and it shouldn't be an issue in the new pane layouts, once we get there.

Also, do note that the new *.layout.tsv (and *.layout) files do not really need the *.paradigm files, as the `*` (or `=`) can be unequivocally replaced by `{{ lemma }} `(and nothing else).